react/OrderItems.tsx

```diff
@@ -217,14 +223,37 @@ export const OrderItemsProvider: FC = ({ children }) => {
     [updateQuantity]
   )
 
+  const addItemToCart = async (items: Partial<Item>[]) => {
+    const task = async () => {
+      try {
+        const {
+          data: { addToCart: newOrderForm },
+        } = await addToCart({ variables: { items } })
+
+        return newOrderForm
+      } catch (error) {
+        console.error(error)
+        return null
+      }
+    }
+
+    const newOrderForm = await enqueue(task)
+
+    newOrderForm && setOrderForm(newOrderForm)
```

Comment:
Use `enqueueTask`. It enqueues the task and sets the orderForm. You will also need it when you implement the feature of merging `addToCart` tasks in the queue.